src/plots/linePlot.ts

```diff
@@ -3,6 +3,7 @@
 module Plottable {
 export module Plots {
   export class Line<X> extends XYPlot<X, number> {
+    private _interpolationFunction: string | ((points: number[][]) => string) = "linear";
```

Comment:
Do you think it makes sense to type it like this instead
https://github.com/palantir/plottable/blob/develop/typings/d3/d3.d.ts#L2176

It's not really consistent with the rest of our codebase